client_plugin/vmdk_plugin/sanity_test.go

```diff
@@ -86,7 +87,7 @@ func init() {
 		"conf_file":                *configFile,
 		"using_conf_file_defaults": usingConfigFileDefaults,
 	}).Info("VMDK plugin tests started ")
-	log.SetFormatter(new(VmwareFormatter))
+	log.SetFormatter(new(log_formatter.VmwareFormatter))
```

Comment:
Nice catch of this one. This looks like a duplicated call even before this PR. I will fix it by this chance.